tests/console/zziplib.pm

```diff
@@ -21,7 +21,8 @@ use registration qw(cleanup_registration register_product add_suseconnect_produc
 use version_utils 'is_sle';
 
 sub run {
-    my $filezip = "files.zip";
+    my $repo_name = "SDK";
+    my $filezip   = "files.zip";
```

Comment:
Both variables would make sense if you would crate method/function from it.

Reply:
$filezip = "files.zip" is not from my code, so I don't know (marcelo wrote this test), but $repo_name is being used to add the repository as well to remove the sdk repository at end.
zypper_call "rr $repo_name";

Reply:
yes but there is only sdk, there can't be anything else :)